blockmanager.go

```diff
@@ -2680,9 +2680,7 @@ func (b *blockManager) handleHeadersMsg(hmsg *headersMsg) {
 
 	// If not current, request the next batch of headers starting from the
 	// latest known header and ending with the next checkpoint.
-	if b.cfg.ChainParams.Net == chaincfg.SimNetParams.Net ||
-		b.cfg.ChainParams.Net == chaincfg.RegressionNetParams.Net ||
-		!b.BlockHeadersSynced() {
+	if isDevNetwork(b.cfg.ChainParams.Net) || !b.BlockHeadersSynced() {
```

Comment:
nit: I dont think this one is necessary. Looks like this is only done for unit testing purposes.

Reply:
I reverted this one back to just simnet, so no changes on blockmanager.go anymore.